components/molecules/Tooltip.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+import { Tooltip as ReactTooltip } from "react-tooltip";
+import arrow from "@/public/arrow.svg";
+import { usePokemonDescription } from "@/hooks/usePokemonDescription";
+
+type Props = {
+  name: string;
+  imageUrl: string;
+};
+export const Tooltip = ({ name, imageUrl }: Props) => {
+  const { description } = usePokemonDescription(name);
+
+  return (
+    <div className="mt-4">
```

Comment:
Good approach, my only recommendation here would be regarding to the effect of the tooltip, adding some border shadow would be better to see the tooltip like hover effect and do not get lost in the actual ItemCard. 👍🏽